src/server.cpp

```diff
@@ -240,6 +240,22 @@ Server::Server(
 
 	if (!gamespec.isValid())
 		throw ServerError("Supplied invalid gamespec");
+
+#if USE_PROMETHEUS
+	m_metrics_backend = std::unique_ptr<MetricsBackend>(new PrometheusMetricsBackend());
+#else
+	m_metrics_backend = std::unique_ptr<MetricsBackend>(new MetricsBackend());
+#endif
+
+	m_uptime_gauge = m_metrics_backend->addGauge("core_server_uptime", "Server uptime (in seconds)");
```

Comment:
this should be a counter - it increases and only ever resets

Reply:
fine